rastervision_pytorch_backend/rastervision/pytorch_backend/pytorch_semantic_segmentation_config.py

```diff
@@ -21,6 +21,7 @@ def get_learner_config(self, pipeline):
             img_channels=pipeline.dataset.img_channels,
             img_format=pipeline.img_format,
             label_format=pipeline.label_format,
+            num_workers=self.num_workers,
```

Comment:
I think this change should be made to the other learner configs (eg.  in pytorch_object_detection_config.py). Otherwise, it would be set by the user, but then wouldn't be used for other tasks.